packages/rx_bloc_cli/example/docs/continuous_delivery.md

```diff
@@ -14,12 +14,8 @@ The `create_distribution` command will populate the directory with the file stru
 The generated `README` file contains steps on where to download and how to name the required files.
 Within the `android` and `ios` directories, paste the previously downloaded files and rename them accordingly.
 
-You should also set the `MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD` variable in your terminal.
-This password will be used for encrypting/decrypting your files within the newly created repository.
-
-```
-export MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD=<PASSWORD>
-```
+Make sure that the `MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD` environment variable is present.
```

Comment:
As discussed the prompt will set the `MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD` environment variable so there is not need of any action item from the engineers. The only place where we need to mention MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD is the actual Github Secret